src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,178 @@
+package kitchenpos.application;
+
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.dao.ProductDao;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Transactional
+@SpringBootTest
+class MenuServiceTest {
+
+    private final String MENU_NAME = "테스트 메뉴";
+    private final Long WRONG_ID = -1L;
```

Comment:
```suggestion
    private static final String MENU_NAME = "테스트 메뉴";
    private static final Long WRONG_ID = -1L;
```

상수 처리를 위한 거라면 static이 붙어도 되겠군요..!

Reply:
헉ㄱ 놓치고 못붙였네요 감사합니다!!